concepticondata/conceptlists/Lynott-2009-423.tsv

```diff
@@ -0,0 +1,424 @@
+Lynott-2009-423-195	195	insipid	TASTELESS	1771	0.285714285714286	1.71428571428571	0.428571428571429	0.285714285714286	1.42857142857143	0.857142857142857	1.60356745147455	0.786795792469443	0.755928946018454	1.2724180205607	1.21498579258791	Visual	0.303030303030303	1	0.693147180559945	7
+Lynott-2009-423-196	196	itchy	ITCHY	149	1	3.14285714285714	4.80952380952381	1.28571428571429	0.285714285714286	0.333333333333333	1.49284005458436	0.511766315719159	1.38357714432037	0.560611910581388	0.577350269189626	Haptic	0.458937198067633	1	0.693147180559945	5
+Lynott-2009-423-197	197	jagged			1	4.71428571428571	4.76190476190476	0.619047619047619	0.238095238095238	0.476190476190476	0.560611910581388	0.436435780471984	1.07126982951031	0.768424485864545	0.928388260322567	Haptic	0.418502202643172	3	1.38629436111989	6
+Lynott-2009-423-198	198	jammy			1	3.33333333333333	1.52380952380952	0.238095238095238	2.47619047619048	4.71428571428571	1.39044357430761	1.56904581255767	0.768424485864545	1.60059512741504	0.717137165600636	Gustatory	0.364341085271318	0	0	5
+Lynott-2009-423-199	199	jingling			0.952380952380952	2.95	1.1	4.95	0.15	0.15	1.356271980176	1.25236618152662	0.223606797749978	0.670820393249937	0.670820393249937	Auditory	0.516129032258065	0	0	8
+Lynott-2009-423-200	200	juicy			1	3.61904761904762	2.19047619047619	1.42857142857143	2.80952380952381	4.85714285714286	1.28359613882991	1.66189793967763	1.80475562255472	1.77817455889594	0.358568582800317	Gustatory	0.230031948881789	2	1.09861228866811	5
+Lynott-2009-423-201	201	khaki			0.952380952380952	4.95	0.55	0.3	0.25	0.1	0.223606797749978	1.43178210632764	0.978720969859186	0.786397515657049	0.447213595499958	Visual	0.788617886178862	2	1.09861228866811	5
+Lynott-2009-423-202	202	large	BIG	1202	1	4.85714285714286	4.19047619047619	1.47619047619048	0.380952380952381	0.571428571428571	0.358568582800317	1.40068010691405	1.60059512741504	1.2440333788203	1.46872150428284	Visual	0.390041493775934	337	5.82304589548302	5
+Lynott-2009-423-203	203	laughing			1	4.33333333333333	0.55	4.71428571428571	0.047619047619048	0.047619047619048	0.912870929175277	1.09904264559757	0.462910049886275	0.218217890235992	0.218217890235992	Auditory	0.481454188160157	26	3.29583686600433	8
+Lynott-2009-423-204	204	leathery			1	4.14285714285714	4.57142857142857	0.523809523809524	2.85714285714286	0.380952380952381	0.654653670707977	0.507092552837111	1.20909253653505	1.35224680756563	0.920662287496912	Haptic	0.33587786259542	1	0.693147180559945	8
+Lynott-2009-423-205	205	lemony			1	2.9047619047619	0.714285714285714	0	3.71428571428571	4.95238095238095	1.54611648670991	0.956182887467515	0	1.00711752754369	0.218217890235994	Gustatory	0.403100775193798	0	0	6
+Lynott-2009-423-206	206	light	LIGHT (WEIGHT)	1052	1	4.14285714285714	2.38095238095238	1	1	1.57142857142857	1.19522860933439	2.06097526613471	1.51657508881031	1.44913767461894	1.6604646509766	Visual	0.311320754716981	216	5.37989735354046	5
```

Comment:
Could also refer to the meaning 'bright'.

Reply:
I'd suggest to include an additional concept for LIGHT (BRIGHTNESS) because it is also one of the frequent meanings in the Collins dictionary.

Reply:
I just saw that we have LIGHT (COLOR) as a concept: "(Of color) having a relatively small amount of color components." Would this fit better because the dominant modality is visual and if light has less color components it is bright?

Reply:
Yes, seems like a reasonable choice.